packages/search-in-workspace/src/browser/search-in-workspace-widget.tsx

```diff
@@ -367,6 +367,7 @@ export class SearchInWorkspaceWidget extends BaseWidget implements StatefulWidge
     protected renderReplaceAllButtonContainer(): React.ReactNode {
         return <div className='replace-all-button-container'>
             <span
+                title='Replace All (Submit Search to Enable)'
```

Comment:
What do you mean: "Submit Search to Enable"  ?
Also, do we have a key bindings like vscode to "Replace all"

Reply:
`vscode` has `Replace All (Submit Search to Enable)` since the button is disabled without any search terms.

Reply:
Tested. I have the same question.

Reply:
It only means that if there is no search term, and replace term, then the button is disabled. 
I'll keep the tooltip generic to only `Replace All`

Reply:
What about the second bullet , vscode has a keybinding for the "Replace all" button. Does Theia has one ?

Reply:
@lmcbout I don't believe so, I have yet to see any of our widgets contribute a keybinding.